inst/templates/methods-adcc/adcc-lab-methods.Rmd

```diff
@@ -0,0 +1,25 @@
+---
+title: "lab-methods"
+date: "2022-10-26"
+---
+
+<!-- This file was adapted from VISCtemplates {{pkg_ver}}. -->
+
+
+# Lab Methods GTL
+
+The qualified GranToxiLux Antibody-Dependent Cell-Mediated Cytotoxicity (GTL-ADCC) assay was performed as previously described [@Pollara2014]. Target cells were a clonal isolate of the CEM.NKRCCR5 CD4+ T-cell line (NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH: from Dr. Alexandra Trkola [@Trkola1999]. These cells were  coated with recombinant gp120s representing the HIV-1 envelopes of the subtype [specify subtype and antigens, e.g. C (TV1 and 1086c)]. Effector cells were PBMCs obtained from a HIV-seronegative donor with heterozygous for Fc$\gamma$R3A at position 158 (158F/V). PBMCs were obtained by leukapheresis to collect enough cells for completion of the study with a single donation, minimizing potential effector cell population variability effects on the study outcome.  PBMCs were used at an effector cell to target cell ratio of 30:1. Serum samples were tested after five-fold serial dilutions starting at 1:50 (1:50, 1:250, 1:1250, 1:6250, 1:31250, and 1:156250). Each plate has one standardized positive control in duplicate and one standardized negative control in duplicate. 
+
+ADCC is quantified as net percent granzyme B activity, which is the percent of target cells positive for GTL (an indicator of granzyme B uptake) minus the percent of target cells positive for GTL when incubated with effector cells in the absence of a source of antibodies. Flow cytometry is used to quantify the frequency of granzyme B positive cells.
+
+# Lab Methods Luciferase
+
+We utilized a modified version of a previously published ADCC luciferase procedure [@Pollara2014, @Fisher2019] . Briefly, CEM.NKRCCR5 cells [@Trkola1999] were used as targets for ADCC luciferase assays after infection by one of the following HIV-1 [vaccine-matched, if all IMCs are vaccine-matched; if not, indicate match in table below] IMCs:
```

Comment:
separate multiple references in same citation with a semicolon, not comma

Reply:
Great catch! I've updated in commit 90fa50e